tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "target": "es6",
+        "target": "es2018",
```

Comment:
_No real review yet.. just since I stumbled over this change_

I am not sure but I think we used to have issues with the browser-bundle when increasing the target version.. I might be wrong though...